main.c

```diff
@@ -1077,6 +1077,9 @@ int main(int argc, char **argv)
 	}
 
 	tcmu_dbg("handler path: %s\n", handler_path);
+
+	tcmu_reset_netlink();
```

Comment:
I think we need to do this in 2 or 3 stages for the restart case, because we could have apps sending requests while tcmu-runner is reinitializing. If we block, reset and unblock here, apps could send new requests after the unblock and they would be stuck because we have not opened the nl socket yet.

So block nl here first then unblock when the nl socket is setup and the daemon is fully ready to handle nl requests. I think that should be right before we call g_main_loop_run. We would then want to run the reset nl call any time between the block and unblock.